python_examples/megno/problem.py

```diff
@@ -32,7 +32,7 @@ def setup_planet(com, mass, period, M, omega, eccentricity):
     if eccentricity>0.8:
         E = math.pi
     F = E - eccentricity*math.sin(M) - M
-    for i in xrange(100):
+    for i in range(100):
```

Comment:
Well, `range` is also much slower for large N compared to `xrange` (as `range` initializes the entire array first and the iterates over it). But that might not matter for most people (it certainly doesn't in the above line).